### ISSUE TITLE
how is the scribe port exposed?

### ISSUE THREAD

**Issue by raam86** [DealerDotCom/zipkin-elasticbeanstalk#2]

**Comment by devinsba** [comment]
This is more of an example of getting the minimum working. We don't use scribe so I didn't spend any time trying to configure it. It shouldn't be difficult to add

**Comment by codefromthecrypt** [comment]
underneath, scribe is still on by default. so I think you're asking how to get to port 9410?

**Comment by raam86** [comment]
Yes, after spending quite some time on it seems like elasticbeanstalk ignores all but the first port declared in the Dockerfile

**Comment by ryangardner** [comment]
You have to use an ebextension to add a second elb that exposes the second
port

**Comment by raam86** [comment]
That didn't seem to work since only the first port is read by ELB. Moved deployment to ECS and it's pretty good.
